linera-views/src/backends/rocks_db.rs

```diff
@@ -468,6 +590,20 @@ pub async fn create_rocks_db_test_store() -> RocksDbStore {
         .expect("store")
 }
 
+/// Creates a RocksDB database client to be used for benchmarks.
+/// The temporary directory has to be carried because if it goes
+/// out of scope then the RocksDB client can become unstable.
+#[cfg(with_testing)]
+pub async fn create_rocks_db_benchmark_store() -> RocksDbStore {
```

Comment:
Perhaps for another PR but throughout `linera-views` we have many such functions that are basically constructors and should be static methods.

Reply:
Ok, will be done.